runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8320,210 +8320,210 @@ done:;
+		JUMP_TABLE_ENTRY(JBdup2x1), /* 93 */
+		JUMP_TABLE_ENTRY(JBdup2x2), /* 94 */
+		JUMP_TABLE_ENTRY(JBswap), /* 95 */
+		JUMP_TABLE_ENTRY(JBiadd), /* 96 */
+		JUMP_TABLE_ENTRY(JBladd), /* 97 */
+		JUMP_TABLE_ENTRY(JBfadd), /* 98 */
+		JUMP_TABLE_ENTRY(JBdadd), /* 99 */
+		JUMP_TABLE_ENTRY(JBisub), /* 100 */
+		JUMP_TABLE_ENTRY(JBlsub), /* 101 */
+		JUMP_TABLE_ENTRY(JBfsub), /* 102 */
+		JUMP_TABLE_ENTRY(JBdsub), /* 103 */
+		JUMP_TABLE_ENTRY(JBimul), /* 104 */
+		JUMP_TABLE_ENTRY(JBlmul), /* 105 */
+		JUMP_TABLE_ENTRY(JBfmul), /* 106 */
+		JUMP_TABLE_ENTRY(JBdmul), /* 107 */
+		JUMP_TABLE_ENTRY(JBidiv), /* 108 */
+		JUMP_TABLE_ENTRY(JBldiv), /* 109 */
+		JUMP_TABLE_ENTRY(JBfdiv), /* 110 */
+		JUMP_TABLE_ENTRY(JBddiv), /* 111 */
+		JUMP_TABLE_ENTRY(JBirem), /* 112 */
+		JUMP_TABLE_ENTRY(JBlrem), /* 113 */
+		JUMP_TABLE_ENTRY(JBfrem), /* 114 */
+		JUMP_TABLE_ENTRY(JBdrem), /* 115 */
+		JUMP_TABLE_ENTRY(JBineg), /* 116 */
+		JUMP_TABLE_ENTRY(JBlneg), /* 117 */
+		JUMP_TABLE_ENTRY(JBfneg), /* 118 */
+		JUMP_TABLE_ENTRY(JBdneg), /* 119 */
+		JUMP_TABLE_ENTRY(JBishl), /* 120 */
+		JUMP_TABLE_ENTRY(JBlshl), /* 121 */
+		JUMP_TABLE_ENTRY(JBishr), /* 122 */
+		JUMP_TABLE_ENTRY(JBlshr), /* 123 */
+		JUMP_TABLE_ENTRY(JBiushr), /* 124 */
+		JUMP_TABLE_ENTRY(JBlushr), /* 125 */
+		JUMP_TABLE_ENTRY(JBiand), /* 126 */
+		JUMP_TABLE_ENTRY(JBland), /* 127 */
+		JUMP_TABLE_ENTRY(JBior), /* 128 */
+		JUMP_TABLE_ENTRY(JBlor), /* 129 */
+		JUMP_TABLE_ENTRY(JBixor), /* 130 */
+		JUMP_TABLE_ENTRY(JBlxor), /* 131 */
+		JUMP_TABLE_ENTRY(JBiinc), /* 132 */
+		JUMP_TABLE_ENTRY(JBi2l), /* 133 */
+		JUMP_TABLE_ENTRY(JBi2f), /* 134 */
+		JUMP_TABLE_ENTRY(JBi2d), /* 135 */
+		JUMP_TABLE_ENTRY(JBl2i), /* 136 */
+		JUMP_TABLE_ENTRY(JBl2f), /* 137 */
+		JUMP_TABLE_ENTRY(JBl2d), /* 138 */
+		JUMP_TABLE_ENTRY(JBf2i), /* 139 */
+		JUMP_TABLE_ENTRY(JBf2l), /* 140 */
+		JUMP_TABLE_ENTRY(JBf2d), /* 141 */
+		JUMP_TABLE_ENTRY(JBd2i), /* 142 */
+		JUMP_TABLE_ENTRY(JBd2l), /* 143 */
+		JUMP_TABLE_ENTRY(JBd2f), /* 144 */
+		JUMP_TABLE_ENTRY(JBi2b), /* 145 */
+		JUMP_TABLE_ENTRY(JBi2c), /* 146 */
+		JUMP_TABLE_ENTRY(JBi2s), /* 147 */
+		JUMP_TABLE_ENTRY(JBlcmp), /* 148 */
+		JUMP_TABLE_ENTRY(JBfcmpl), /* 149 */
+		JUMP_TABLE_ENTRY(JBfcmpg), /* 150 */
+		JUMP_TABLE_ENTRY(JBdcmpl), /* 151 */
+		JUMP_TABLE_ENTRY(JBdcmpg), /* 152 */
+		JUMP_TABLE_ENTRY(JBifeq), /* 153 */
+		JUMP_TABLE_ENTRY(JBifne), /* 154 */
+		JUMP_TABLE_ENTRY(JBiflt), /* 155 */
+		JUMP_TABLE_ENTRY(JBifge), /* 156 */
+		JUMP_TABLE_ENTRY(JBifgt), /* 157 */
+		JUMP_TABLE_ENTRY(JBifle), /* 158 */
+		JUMP_TABLE_ENTRY(JBificmpeq), /* 159 */
+		JUMP_TABLE_ENTRY(JBificmpne), /* 160 */
+		JUMP_TABLE_ENTRY(JBificmplt), /* 161 */
+		JUMP_TABLE_ENTRY(JBificmpge), /* 162 */
+		JUMP_TABLE_ENTRY(JBificmpgt), /* 163 */
+		JUMP_TABLE_ENTRY(JBificmple), /* 164 */
+		JUMP_TABLE_ENTRY(JBifacmpeq), /* 165 */
+		JUMP_TABLE_ENTRY(JBifacmpne), /* 166 */
+		JUMP_TABLE_ENTRY(JBgoto), /* 167 */
 		JUMP_TABLE_ENTRY(JBunimplemented),
```

Comment:
please number the unimplemented ones as well

Reply:
Sorry for that, I thought I should leave it empty. 
I have modified it and numbered JBunimplemented ByteCode